mbed_app.json

```diff
@@ -1,8 +1,6 @@
 {
-    "requires": ["bare-metal"],
     "target_overrides": {
       "*": {
-        "target.c_lib": "small",
```

Comment:
Why removing this?

Reply:
I just now realized and reverting this changes